staging/src/k8s.io/client-go/util/retry/util.go

```diff
@@ -78,7 +65,38 @@ func OnError(backoff wait.Backoff, errorFunc func(error) bool, fn func() error)
 	return err
 }
 
-// RetryOnConflict executes the function function repeatedly, retrying if the server returns a conflicting
+// RetryOnConflict is used to make an update to a resource when you have to worry about
+// conflicts caused by other code making unrelated updates to the resource at the same
+// time. fn should fetch the resource to be modified, make appropriate changes to it, try
+// to update it, and return (unmodified) the result from the update function. On a
```

Comment:
> (unmodified) the result

The "result" is the "error" returned by the update, right? If so, can we just say the "the (unmodified) error" to be more specific?